### PR TITLE
The welcome text in the navbar looks better

### PR DESCRIPTION
**Pull request by AlexaBus**
Deleted ThymeleafConfig as it was not needed and added functionality tocheck user name against the db at login

Changed the password encrypter to use spring bcrypt

1. In TheatreSecurityConfiguration the getPasswordEncoder uses the same spring BCrypt method
2. just some empty lines in LoginController
3. removed register.js as the e-mail validation is done in the html and i moved the password matching javascript in the html also as there were only a few lines
4. Now, when trying to login, if bad credentials are used the message Invalid username or password is shown
5. register.html now contains the js for password checking

### CODE REVIEW

File: otheatre/src/main/resources/templates/register.html [hunk 1]
```diff
@@ -51,7 +51,17 @@ <h3 class="panel-title">Orange Theatre Registration
                             <div class="col-xs-6 col-sm-6 col-md-6">
                                 <div class="form-group">
                                     <input type="password" name="password_confirmation" id="password_confirmation"
-                                           class="form-control input-sm" placeholder="Confirm Password">
+                                           class="form-control input-sm" placeholder="Confirm Password" oninput="check(this)" required>
+                                    <script language='javascript' type='text/javascript'>
+                                            function check(input) {
+                                                if (input.value != document.getElementById('password').value) {
+                                                    input.setCustomValidity('Confirmation password is not the same.');
+                                                } else {
```
Comment (emanueldumitru) [inline otheatre/src/main/resources/templates/register.html]: Asta cred ca ar trebuie sa fie intr-un fisier separat de js si <script th:src="..."></script> sa il pui la final inainte de  tagul </body>, nu aici, as a best practice.

Comment (AlexaBus) [reply otheatre/src/main/resources/templates/register.html]: original, era intr-un fisier separat. dar mi s-a parut prea putin cod ca sa mai tin fisierul doar pentru asta
vrei sa refac fisierul inainte de merge?

Comment (emanueldumitru) [reply otheatre/src/main/resources/templates/register.html]: da, nu conteaza cat de mic e fisierul, nu se incorporeaza script-uri, nu e elegant

Comment (AlexaBus) [reply otheatre/src/main/resources/templates/register.html]: ok. refac si revin

**Comment by AlexaBus** [comment]
@Emanuel9 am refacut fisierul js si daca tot am modificat am facut si partea de login direct dupa register
am facut si squash astfel incat sa fie un singur commit

**Pull request merged**